src/networking/l7-example.yaml

```diff
@@ -1,9 +1,9 @@
 apiVersion: "cilium.io/v2"
 kind: CiliumNetworkPolicy
+description: "Allow HTTP GET /public from env=prod to app=service"
```

Comment:
This won't work in cilium 1.11 anymore. I think you forgot to rebase from master. See https://github.com/hlesey/k8s-labs/pull/17

Reply:
reverted